src/verify.integration.test.ts

```diff
@@ -2,324 +2,268 @@
+import { documentRopstenValidWithToken } from "../test/fixtures/documentRopstenValidWithToken";
+import { documentRopstenRevokedWithToken } from "../test/fixtures/documentRopstenRevokedWithToken";
 
 describe("verify(integration)", () => {
-  it("returns false if document's hash is invalid and was not issued", async () => {
-    const results = await verify(documentTampered, "ropsten");
+  it("should for OpenAttestationTamperCheck and OpenAttestationEthereumDocumentStoreIssued when document's hash is invalid and was not issued", async () => {
+    const results = await defaultVerificationManager(documentTamperedWithCertificateStore, {
+      network: "ropsten"
+    });
 
-    expect(results).toEqual(
-      expect.objectContaining({
-        hash: { checksumMatch: false },
-        issued: {
-          issuedOnAll: false,
+    expect(results).toStrictEqual([
+      {
+        data: {
+          checksumMatch: false
+        },
+        message: "Certificate has been tampered with",
+        status: "INVALID",
+        type: "OpenAttestationTamperCheck"
+      },
+      {
+        data: {
           details: [
             {
               address: "0x20bc9C354A18C8178A713B9BcCFFaC2152b53990",
-              error: expect.stringMatching("exception"),
+              error:
+                'call exception (address="0x20bc9C354A18C8178A713B9BcCFFaC2152b53990", method="isIssued(bytes32)", args=["0x85df2b4e905a82cf10c317df8f4b659b5cf38cc12bd5fbaffba5fc901ef0011b"], version=4.0.40)',
               issued: false
             }
-          ]
+          ],
+          issuedOnAll: false
         },
-        revoked: {
-          revokedOnAny: false,
+        message: "Certificate has not been issued",
+        status: "INVALID",
+        type: "OpenAttestationEthereumDocumentStoreIssued"
+      },
+      {
+        data: {
           details: [
             {
               address: "0x20bc9C354A18C8178A713B9BcCFFaC2152b53990",
               revoked: false
             }
-          ]
+          ],
+          revokedOnAny: false
         },
-        valid: false
-      })
-    );
-  });
-
-  it("returns true if Mainnet document is valid", async () => {
-    const results = await verify(documentMainnetValid);
-
-    expect(results).toEqual({
-      hash: { checksumMatch: true },
-      issued: {
-        issuedOnAll: true,
-        details: [
-          {
-            address: "0x007d40224f6562461633ccfbaffd359ebb2fc9ba",
-            issued: true
-          }
-        ]
-      },
-      revoked: {
-        revokedOnAny: false,
-        details: [
-          {
-            address: "0x007d40224f6562461633ccfbaffd359ebb2fc9ba",
-            revoked: false
-          }
-        ]
+        status: "VALID",
+        type: "OpenAttestationEthereumDocumentStoreRevoked"
       },
-      valid: true
-    });
+      {
+        message: 'Document issuers doesn\'t have "documentStore" or "token" property',
+        status: "SKIPPED",
```

Comment:
Just a question... 

If I have the `OpenCertsRegistry` extension:
1. This method will show skipped
2. OpenCertsRegistry method will show valid

How should I go about handling it in the method `isValid`?